pico/tools/src/main/java/io/helidon/pico/tools/CommonUtils.java

```diff
@@ -24,14 +24,16 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.List;
-import java.util.Objects;
+import java.util.Set;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 
 /**
  * General utils.
  */
-final class CommonUtils {
+public final class CommonUtils {
```

Comment:
We usually don't want to expose util classes to the public API, can we make it package private please?